Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = [ "rtc_tenclave", "rtc_uenclave", "rtc_types", "rtc_data_enclave", "rtc_data_service" ] 
+members = [ "rtc_tenclave", "rtc_uenclave", "rtc_types", "rtc_data_enclave", "rtc_data_service" , "rtc_app_backend"] 
```

Comment:
This should probably be reverted?

Reply:
I noticed when I was building now. I made the change. I will push.